vtex-apps/checkout-summary
# Remove Go To Checkout button

#### What problem is this solving?

We're moving the "Go to checkout" button to `checkout-cart` in order to place it on a sticky footer (see https://github.com/vtex-apps/checkout-cart/pull/23).

#### How should this be manually tested?
The test plan of https://github.com/vtex-apps/checkout-cart/pull/23 applies to this PR as well. 

Alternatively, you can add [some item](https://button--vtexgame1.myvtex.com/teste-so-delivery-copy-256--copy-257--copy-261--copy-262-/p) to your cart, go to `/cart` and verify the "Checkout" button is gone.

#### Checklist/Reminders

- [x] [Linked this PR to a Clubhouse story (if applicable)](https://app.clubhouse.io/vtex/story/24232/implementar-footer-fixo-no-s-do-carrinho).

#### Screenshots or example usage

#### Type of changes


✔️ | Type of Change
---|---
_ | Bug fix 
_ | New feature 
✔️ | Breaking change 
✔️ | Technical improvements 

#### Notes
